examples/1-execute-query-mutation/src/App.re

```diff
@@ -1,18 +1,16 @@
-module Styles = {
-  open Css;
-
-  let page =
-    style([
-      position(absolute),
-      top(px(0)),
-      bottom(px(0)),
-      left(px(0)),
-      right(px(0)),
-      padding(px(20)),
-    ]);
-};
-
 [@react.component]
 let make = () => {
-  <div className=Styles.page> <ColdStart /> <Previewer /> </div>;
+  <div
```

Comment:
All the style changes basically boil down to 1) If it's a one-off style, just use `ReactDOMRe.Style`, and 2) If the styles are more extensive, use `bs-css` with a separate `module` for styles.